tests/framework/installer/ceph_manifests_v1.1.go

```diff
@@ -169,6 +169,14 @@ spec:
                   type: integer
             placement: {}
             resources: {}
+            cleanupPolicy:
```

Comment:
actually we don't want to update the ceph_manifests_v1.1.go. We want that to remain in the same state as v1.1.